src/js/languages/default.json

```
@@ -80,7 +83,9 @@
       "plural": "{numOfFiles} files are too large. Select files which are no larger than {maxSize}."
     },
     "remove": "remove",
-    "removeAll": "remove all"
+    "removeAll": "remove all",
+    "removeMaxFileSize": "Error - remove file: {fileName} (exceeds max file size) of {maxSize}.",
+    "removeMaxFileCount": "Error - remove file: {fileName}, (maximum number of files) - {maxNumber} exceeded."
```

It seems odd to put this message in the remove button. I think the remove button should just say "Remove file: {fileName}". Shouldn't the validation error text be elsewhere, like the alert icon (ie. tells you what the alert icon means")

```suggestion
    "removeMaxFileSize": "Remove file: {fileName}. This file exceeds max file size of {maxSize}",
    "removeMaxFileCount": "Remove file: {fileName}. This file exceeds the maximum number of files) - {maxNumber}"
```

---

The word `Error` was used for comparison with the [draft PR ](https://github.com/grommet/grommet/pull/7674/files#diff-ecd05461ba0fd4b2a00e66e9708f8d8a59bf3dde54716424f8aadaa0a4750430)that Bill will be testing against, the draft PR has a better message that is concise and direct. Please see below and let me know what you think

```
"removeMaxFileSize": "Remove file: {fileName}. This file is too large. Maximum allowed size is {maxSize}.",
"removeMaxFileCount": "Remove file: {fileName}. Too many files uploaded. Maximum allowed is {maxNumber}."
```

---

Better thanks. I still think it's not right that it's on the remove button and should be instead on the alert icon (ie. why I'm getting this icon). Testing with Bill will help